frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -158,19 +158,19 @@ pods:
         readiness-check:
           cmd: >
             export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/) &&
-          {{#KERBEROS_ENABLED}}
-            KRB5_CONFIG={{JRE_VERSION}}/lib/security/krb5.conf kinit -k -t $MESOS_SANDBOX/hdfs.keytab $KERBEROS_PRIMARY/$TASK_NAME.$FRAMEWORK_HOST@$KERBEROS_REALM &&
-          {{/KERBEROS_ENABLED}}
+          {{#SECURITY_KERBEROS_ENABLED}}
+            KRB5_CONFIG={{JRE_VERSION}}/lib/security/krb5.conf kinit -k -t $MESOS_SANDBOX/hdfs.keytab $SECURITY_KERBEROS_PRIMARY/$TASK_NAME.$FRAMEWORK_HOST@$SECURITY_KERBEROS_REALM &&
```

Comment:
Just a question for myself. Does HDFS not support JAAS, or does it require that `kinit` be used explicitly.

Reply:
It actually doesn't require it, as JAAS is also supported.